samples/streamlit-in-snowflake/sis-deploy/README.md

```diff
@@ -37,28 +76,68 @@ warehouse = "<WAREHOUSE>"
 
 # Set up local environment
 
+Because streamlit-in-snowflake uses conda to install and packages, it's best to install
+conda locally and use that. These are defined in environment.yml
+
+```yml
+name: sf_env
+channels:
+  - snowflake
+dependencies:
+  - python=3.8.13 # The latest python available in snowflake as of 2024-02-12
+  - streamlit=1.26.0 # The latest streamlit available in snowflake as of 2024-02-12
```

Comment:
so streamlit is 1.22 in SiS for most customers right now, we should be going into a public preview next week so I think this is fine

Reply:
Ah, good catch. Should I update the comment to reflect the date, or is this fine as-is?

Reply:
nah it is fine as is I think